packages/cmf-webpack-plugin/src/index.js

```diff
@@ -27,17 +27,26 @@ function ReactCMFWebpackPlugin(options = {}) {
 	};
 }
 
+function getCmfconfig(cmfconfigPath) {
+	const cmfconfig = require(cmfconfigPath);
+	if (process.env.CMF_ENV) {
+		return cmfconfig[process.env.CMF_ENV];
+	}
+	return cmfconfig;
+}
```

Comment:
it really looks like a duplicate.

Reply:
cmf-webpack-plugin has cmf as dependency, but cmf also invalidates the require cache in this function - cmf-webpack-plugin does not. It would be a change to cmf-webpack-plugin that I don't know is a good idea or not.